src/assets/wise5/authoringTool/choose-move-node-location/choose-move-node-location.component.html

```diff
@@ -0,0 +1,153 @@
+<ng-template #moveAfterButton let-nodeId="nodeId">
+  <button
+    mat-raised-button
+    color="primary"
+    [disabled]="!canMove(nodeId)"
+    (click)="move(nodeId, true)"
+    matTooltip="Move after"
+    matTooltipPosition="above"
+    i18n-matTooltip
+  >
+    <mat-icon>subdirectory_arrow_left</mat-icon>
+  </button></ng-template
+>
+<ng-template #moveInsideButton let-nodeId="nodeId">
+  <button
+    mat-raised-button
+    color="primary"
+    (click)="move(nodeId, false)"
+    matTooltip="Move as first"
+    matTooltipPosition="above"
+    i18n-matTooltip
+  >
+    <mat-icon>call_received</mat-icon>
+  </button></ng-template
+>
+<ng-template #nodeIconAndTitle let-nodeId="nodeId" let-showPosition="showPosition">
+  <node-icon [nodeId]="nodeId" size="18"></node-icon>&nbsp;
+  <p>
+    <span *ngIf="showPosition">{{ getNodePositionById(nodeId) }}: </span>{{ getNodeTitle(nodeId) }}
+  </p>
+</ng-template>
+<div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px">
+  <h5 i18n>Choose a new location by clicking one of the buttons below</h5>
+  <button mat-raised-button color="primary" routerLink=".." aria-label="Cancel" i18n>Cancel</button>
+</div>
+<div style="margin-top: 20px; margin-left: 20px">
+  <ng-container *ngIf="moveGroup">
+    <ng-container
+      *ngTemplateOutlet="moveInsideButton; context: { nodeId: 'group0' }"
+    ></ng-container>
+  </ng-container>
+  <div
+    *ngFor="let nodeId of nodeIds"
+    [ngClass]="{
+      'node-select--group': isGroupNode(nodeId),
+      'node-select--node': !isGroupNode(nodeId)
+    }"
+  >
+    <div
+      id="{{ nodeId }}"
+      [ngClass]="{
+        groupHeader: isGroupNode(nodeId),
+        stepHeader: !isGroupNode(nodeId),
+        branchPathStepHeader: isNodeInAnyBranchPath(nodeId) && !isGroupNode(nodeId)
+      }"
+      [ngStyle]="{ 'background-color': getBackgroundColor(nodeId) }"
+    >
+      <div fxLayout="row" fxLayoutGap="8px">
+        <div fxLayout="row" fxLayoutAlign="start center">
+          <ng-container
+            *ngTemplateOutlet="nodeIconAndTitle; context: { nodeId: nodeId, showPosition: true }"
+          ></ng-container>
+        </div>
+        <ng-container *ngIf="isGroupNode(nodeId) && !moveGroup">
+          <ng-container
+            *ngTemplateOutlet="moveInsideButton; context: { nodeId: nodeId }"
+          ></ng-container>
+        </ng-container>
+        <ng-container *ngIf="!isGroupNode(nodeId) || (isGroupNode(nodeId) && moveGroup)">
+          <ng-container
+            *ngTemplateOutlet="moveAfterButton; context: { nodeId: nodeId }"
+          ></ng-container>
+        </ng-container>
+      </div>
+    </div>
+  </div>
+</div>
+<div>
+  <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px">
+    <h6 class="underline">Unused Lessons</h6>
```

Comment:
Add i18n to the "Unused Lessons" element.

Reply:
Done.